docs/self-managed/setup/install.md

```diff
@@ -37,7 +37,7 @@ The following charts will be installed as part of Camunda 8 Self-Managed:
   - _Note_: The chart is disabled by default and needs to be [enabled explicitly](#install-web-modeler) as Web Modeler is only available to enterprise customers.
 
 :::note Amazon OpenSearch Helm support
-The existing Helm charts use the Elasticsearch configurations by default and are not yet prepared with the OpenSearch configurations as templates/pre-filled. The Helm charts can still be used to install for OpenSearch, but some adjustments are needed beforehand.
+The existing Helm charts use the Elasticsearch configurations by default. The Helm charts can still be used to connect to AWS-managed OpenSearch. Refer to [using AWS-managed OpenSearch](/self-managed/setup/guides/using-existing-opensearch.md).
```

Comment:
One comment. The AWS product name is "Amazon OpenSearch Service". Could we update AWS-managed to Amazon OpenSearch Service so we are precise in the name?

Reply:
Great catch @theburi 👍

Reply:
I would have to make this name change in the other docs as well. I can do that.